pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/OncHistoryTestUtil.java

```diff
@@ -104,6 +104,19 @@ public ParticipantDto createParticipant(String guid, DDPInstanceDto ddpInstanceD
         return testParticipant;
     }
 
+    public ParticipantDto createSharedLearningParticipant(String guid, DDPInstanceDto ddpInstanceDto, String dob) {
```

Comment:
Could we move this to `TestParticipantUtil`?  Since it's creating a participant, it should probably go in a class that deals with participants.